services/community-backend/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:14
+WORKDIR /app
+
+ENV PATH /app/node_modules/.bin:$PATH
+
+COPY package.json yarn.lock ./
+COPY services/community-backend/package.json /app/services/community-backend/
+
+COPY package.json yarn.lock /app/
+COPY services/claim-backend/package.json /app/services/claim-backend/
```

Comment:
Don't need the claim here

Reply:
fix. btw in similar situation (duplication) is in claim-backend: https://github.com/Taraxa-project/taraxa-platform/blob/develop/services/claim-backend/Dockerfile#L9-L13